companion/src/firmwares/customfunctiondata.cpp

```diff
@@ -223,7 +230,7 @@ bool CustomFunctionData::isFuncAvailable(const int index)
         ((index == FuncScreenshot) && !IS_HORUS_OR_TARANIS(fw->getBoard())) ||
         ((index >= FuncRangeCheckInternalModule && index <= FuncBindExternalModule) && !fw->getCapability(DangerousFunctions)) ||
         ((index >= FuncAdjustGV1 && index <= FuncAdjustGVLast) && !fw->getCapability(Gvars)) ||
-		((index == FuncDisableTouch) && !IS_HORUS_OR_TARANIS(fw->getBoard()))
+		    ((index == FuncDisableTouch) && !IS_HORUS_OR_TARANIS(fw->getBoard()))
```

Comment:
I think companion formatting is a little different here... just follow the pattern of the lines above and don't indent this line so much.

Reply:
I will deliver the proper format in a moment - for some reason TAB arrived there instead of spaces.

Reply:
Ouch... bad editor! I hope you had words with it ;)